README.md

```diff
@@ -0,0 +1,67 @@
+## CALCULATOR
+> A calculator project built to understand  Javascript.
+
+![Screenshot loading...](assets\screenshot.png)
+
+## Built With
+
+- CSS
+- JAVASCRIPT
+- HTML
+
+## Live Demo
+<a href="https://rawcdn.githack.com/Habidoye97/calculator/857d23beefe1844793574fdeddea71aba78201d2/index.html" target="_blank">Calculator</a>
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- On the project GitHub page, navigate to the main page of the repository [this page](https://github.com/Habidoye97/calculator.git).
+- Under the repository name, locate and click on a green button named `Code`.
+- Copy the project URL as displayed.
+- If you're running Windows Operating System, open your command prompt. On Linux, Open your terminal.
+- Change the current working directory to the location where you want the cloned directory to be made. Leave as it is if the current location is where you want the project to be.
+- Type `git clone`, and then paste the URL you copied in Step 3.<br>
+  `$ git clone https://github.com/Habidoye97/calculator.git` <em>Press Enter key</em><br>
+- Press Enter. Your local copy will be created.
+
+Please Note that you must have github installed on your PC, this can be done
+
+### Prerequisites
+
+- [Git](https://gist.github.com/derhuerst/1b15ff4652a867391f03).
+- Web browser (Chrome/Firefox)
+- [Node](https://nodejs.org/en/)
+- [NPM](https://www.npmjs.com/get-npm)
+
+### Usage
+
+Click on the live demo link and peruse the app.
+
+### Making Changes
+
+- `npm install` to install the dependencies and packages
+- `npm start` to test the app after changes were made
```

Comment:
Running `npm start` does not start your project. It looks like this command is not necessary. You should remove this and add the necessary steps to start your project in a browser.
Hint: you can replace this with how to start your project using live server or the conventional method.